addons/mail/models/mail_thread.py

```diff
@@ -258,15 +260,11 @@ def create(self, vals_list):
         if self._context.get('tracking_disable'):
             return super(MailThread, self).create(vals_list)
 
-        # subscribe uid unless asked not to
-        if not self._context.get('mail_create_nosubscribe'):
-            default_followers = self.env['mail.followers']._add_default_followers(self._name, [], self.env.user.partner_id.ids, customer_ids=[])[0][0]
-            for values in vals_list:
-                message_follower_ids = values.get('message_follower_ids') or []
-                message_follower_ids += [(0, 0, fol_vals) for fol_vals in default_followers]
-                values['message_follower_ids'] = message_follower_ids
-
         threads = super(MailThread, self).create(vals_list)
+        # subscribe uid unless asked not to and avoid to add inactive user such as OdooBot
+        if not self._context.get('mail_create_nosubscribe'): # TODO and self.env.user.active:
```

Comment:
why TODO not done ?

Reply:
I removed this TODO (and related comment part), the check if the user is active do more request for each call and add a no-active user to the followers is harmless and rare (because check in all SQL request).